README.md

```diff
@@ -11,6 +11,7 @@ Lista em ordem alfabética.
 | [![Commerce 10](/companies/commerce10.png?raw=true)](https://www.commerce10.com.br)              | [contato@commerce10.com.br](mailto:contato@commerce10.com.br)                    | 6     | São Paulo-SP<br>Fortaleza-CE<br>Canindé-CE                                                                                                                               | ✓       | 
 | [![Defensoria Pública do Tocantins](/companies/dpeto.png?raw=true)](http://defensoria.to.def.br) |                                                                                  | 15    | Palmas-TO                                                                                                                                                                | ✕       |
 | [![Easy Bills](/companies/easy-bills.png?raw=true)](https://www.easybills.io/?locale=pt-BR)      |                                                                                  | 1     |                                                                                                                                                                          | ✓       |
+| [![Fractal Tecnologia](/companies/fractal.png?raw=true)](https://www.fractaltecnologia.com.br)   | [github.com/fractaltecnologia/jobs](https://github.com/fractaltecnologia/jobs)   | 6     | João Pessoa-PB                                                                                                                                                           | ✓       |
```

Comment:
Thanks a lot, @g13ydson! 😃
Can you please include a logo which has **200px x 40px**?